src/services/bitcoin-rpc.service.ts

```
@@ -134,6 +134,25 @@ export class BitcoinRpcService implements OnModuleInit {
         return result;
     }
 
+    private async loadBlockTemplate(blockHeight: number) {
```

ESLint

[error] Delete `··`

(prettier/prettier)